src/main/java/controller/MainController.java

```diff
@@ -24,10 +22,10 @@
 /**
  * <h1>Main Controller</h1> Accepts input and converts it to commands and action
  * for the model or view. In addition to dividing the application into these
- * components, the model�view�controller design defines the interactions between
+ * components, the model�view�controller design defines the interactions between
```

Comment:
damn eclipse encode :(

Reply:
```suggestion
 * components, the model-view-controller design defines the interactions between
```

Reply:
Now you're the one who comitted this in a weird encoding though :p